src/arch/sparcv9/Mir.zig

```diff
@@ -94,6 +94,16 @@ pub const Inst = struct {
         // TODO add other operations.
         sllx,
 
+        /// A.54 Store Integer
+        /// Those uses the arithmetic_3op field.
```

Comment:
Typo: Either "Those use" (though the doc comment should probably refer to its subject as this/these), or "This uses".

Reply:
Ah, thanks, I'll change the doc comments then.

Reply:
Seems like you missed this one.